src/lexparse/lexer_consts.go

```diff
@@ -0,0 +1,108 @@
+package lexparse
+
+type tokenType int
+
+const (
+  SymbolT tokenType = iota
+
+  StringT tokenType = iota
+  CharT tokenType = iota
+  IntT tokenType = iota
+  FloatT tokenType = iota
+  BoolT tokenType = iota
+  NullT tokenType = iota
+
+  ForT tokenType = iota
+  WhileT tokenType = iota
+  InT tokenType = iota
+  TypeT tokenType = iota
+
+  BinopT tokenType = iota // + - * / ^ % & |
+  ComparisonT tokenType = iota // == != < > <= >=
+  SetterT tokenType = iota // = += -= *= /= ^= %= &= |= ->
+  PrefixT tokenType = iota // ! ~ ?
+  SuffixT tokenType = iota // $ # .
+
+  LParT tokenType = iota
+  RParT tokenType = iota
+  LSquareBrT tokenType = iota
+  RSquareBrT tokenType = iota
+  LCurlyBrT tokenType = iota
+  RCurlyBrT tokenType = iota
+  LAngleT tokenType = iota
+  RAngleT tokenType = iota
+
+  DotT tokenType = iota
+  QMarkT tokenType = iota
+  EllipsisT tokenType = iota
+  CommaT tokenType = iota
+  SemicolonT tokenType = iota
+  ColonT tokenType = iota
+  SubtypeT tokenType = iota
+)
+
+// Go doesn't have sets, which is dumb.
```

Comment:
Luckily we're fixing this :p

Reply:
Lol I wonder what the effect will be on anyone else who looks at this code if I fill it with comments disparaging the programming language I chose to use